webxrlayers-1.bs

```diff
@@ -922,8 +929,8 @@ To <dfn>determine the layout attribute</dfn> using an {{XRTextureType}} |texture
 <div class="algorithm" data-algorithm="determine the maximum scalefactor">
 
 To <dfn>determine the maximum scalefactor</dfn> using an {{XRSession}} |session|, an {{XRWebGLRenderingContext}} |context| and an {{XRLayerLayout}} |layout|, the user agent MUST run the following steps:
-  1. Let |largest width| be the largest width of the [=recommended WebGL texture resolution=] from the |session|'s [=list of views=].
-  1. Let |largest height| be the largest height of the [=recommended WebGL texture resolution=] from the |session|'s [=list of views=].
+  1. Let |largest width| be the largest width of the [=recommended WebGL texture resolution=] from the |session|'s [=list of views=] excluding the [=first-person observer view=].
```

Comment:
I think this should be "excluding secondary views" instead? The core spec has the general concept, the AR module defines the semantics of a specific one, but others can exist.